plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -206,6 +219,38 @@ public Self setConsistency(WriteConsistencyLevel consistency) {
         return self();
     }
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
```

Comment:
I don't think we should put the defaults in javadoc here, it's already on the variable